moesifapi/workers.py

```diff
@@ -65,10 +65,7 @@ def __init__(self, queue, api_client, config, debug):
     def send_events(self, batch_events):
         try:
             logger.debug("Sending events to Moesif")
-            batch_events_api_response = self.api_client.create_events_batch(batch_events)
-            # Update the configuration if necessary
-            etag = batch_events_api_response.get("X-Moesif-Config-ETag")
-            self.config.check_and_update(etag)
+            self.api_client.create_events_batch(batch_events)
```

Comment:
@keyur9 removed e-tag check by mistake or it was already happening in create_events_batch?

Reply:
@praves77 we've scheduled job to fetch config, so we'd get updated config. don't have to use e-tag to determine when to update the config